### ISSUE TITLE
HyperParam Tuning Failed Linear Regression

### ISSUE THREAD

**Issue by d4rk-lucif3r** [d4rk-lucif3r/LuciferML#3]
GridSearchCV Failed 
1) Linear Regression
![](https://i.imgur.com/m3BovDv.png)
2) Stochastic Gradient Descent
![](https://i.imgur.com/vAVN6bk.png)
3) K-Neighbors Regressor
![image](https://i.imgur.com/NqP3j3X.png)
4) Gradient Boost
![](https://i.imgur.com/baWtGhC.png)
